src/test/java/io/confluent/kafkarest/integration/ProducerTest.java

```diff
@@ -163,7 +154,7 @@ public void testProducerConfigOverrides() {
     overrides.setProperty("buffer.memory", "1");
     // Note separate ProducerPool since the override should only be for this test, so
     // getProducerPool doesn't work here
-    ProducerPool pool = new ProducerPool(this.restConfig, this.bootstrapServers, overrides);
+    ProducerPool pool = new ProducerPool(this.restConfig, this.brokerList, overrides);
```

Comment:
I think that fell out of copying from schema-registry as a base. Somehow these classes had diverged. I'm happy with either name.